packages/web-components/fast-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../styles";
+
+export const RadioGroupStyles = css`
+    ${display("flex")} :host {
+        align-items: flex-start;
+        margin: calc(var(--design-unit) * 1px) 0;
+        position: relative;
```

Comment:
I'm not sure you need to set a position of relative here; you can likely remove.

Reply:
removing